lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -601,8 +601,7 @@ def make_mpdboot(self):
                              "(text file with minimal entry 'password=<somesecretpassword>')"), mpdconffn)
 
             with open(mpdconffn, 'w') as mpdconff:
-                mpdconff.write("password=%s" % ''.join(random.choice(string.ascii_uppercase + string.digits)
-                                                       for x in range(10)))
+                mpdconff.write("password=" + ''.join(random.choices(string.ascii_uppercase + string.digits, k=10)))
```

Comment:
why u no f-string? :wink:

Reply:
I don't see the point here, it would only make the code harder to read by adding `{...}`

Reply:
it's faster. 
And I find the f-string notation more readable but that's personal preference I guess.
not going to die on this hill though. :)

Reply:
"faster" makes no sense here as an argument, this password file is only created if it doesn't exist yet, and it'll be done in microseconds using f-strings or not :)

Reply:
sure, but the best way to learn new practices is to use them all the time. circumventing the linter check by doing the `+` is not good practice. :wink: but again, I won't die on this hill.